torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -85,6 +85,7 @@ constexpr int64_t kBasicChannelPriority = 0;
 
 #if TENSORPIPE_HAS_CUDA_IPC_CHANNEL && defined(USE_CUDA_NOT_ROCM)
 constexpr int64_t kCudaIpcChannelPriority = 300;
+constexpr int64_t kCudaXthChannelPriority = 400;
```

Comment:
This shouldn't be guarded by the `TENSORPIPE_HAS_CUDA_IPC_CHANNEL`. That macro only control the CUDA_IPC channel.

Reply:
ah, yep, good catch!